pages/docs/concepts/asyncapi-document/tags.md

```diff
@@ -0,0 +1,249 @@
+
+The `tags` object serves as a container for defining and managing tags in a systematic manner. Within the `tags` object, you can define individual tags, specifying a name and providing an optional description that adds further context to the tag's purpose or usage. 
+
+Once tags are defined within the `tags` object, they can be assigned to channels, operations or any other component using the tags property. By including the relevant tag names in the tags property, objects such as channels, servers or components are associated with the appropriate tags, grouping them based on their shared characteristics.
+
+Additionally, `tags` can also be included in the [`components` object](https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#componentsObject) of an AsyncAPI document, which enables the reusability of the tags. `tags` included in the `components` object can be re-used by [reference objects](https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#referenceObject).
+
+## `Tags` in AsyncAPI Document
+The `tags` object is a list of `tag` objects which can be referenced by [reference object](https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#referenceObject).
+
+The `tags` object within the AsyncAPI specification contains some fields that define and describe individual tags.
+
+The behaviour and impact of the `tags` object within the AsyncAPI document can vary depending on the context of the object in which it is being used. For instance, The way the `tags` object impacts a server object is different from when it is used under the channels object.
+
+Some of the fields that the `tags` object holds are:
+- `name`: The name of the tag.
+- `description`: A short description for the tag.
+- `externalDocs`: Additional external documentation for this tag.
+
+### `Tags` in `Info` Object
+Tags when defined within the `tags` property of the `info` object provides a high-level categorization and classification for the entire AsyncAPI document. These tags that are defined under the `info` object give a global context to the entire application, representing overarching themes or functional areas within the event-driven system. They serve as a way to group objects such as channels or servers based on their broader significance, providing a holistic understanding of the application's structure.
+
+Here's a visual representation of the `tags` object inside an `info` object in an AsyncAPI document:
+```mermaid
+graph LR
+  A[asyncapi] --> B[info]
+  B --> C[title]
+  B --> D[version]
+  B --> E[description]
+  B --> F[tags]
+  F --> G[name]
+  F --> I[description]
+  F --> K[externalDocs]
+```
+
+Below is an example of the `tags` object inside the `info` object in an AsyncAPI document:
+```yaml
+asyncapi: 3.0.0
+info:
+  title: AsyncAPI Documentation
+  version: 1.0.0
+  description: |
+    This AsyncAPI document provides a overview
+    of the event-driven system.
+  tags:
```

Comment:
regarding names of tags and description, always best to copy real life example. Take them from ZoraBots https://bitbucket.org/qbmt/zbos-mqtt-api/src/master/zbos_mqtt-all-asyncapi.json (bottom of the document, of course not all, maybe 2-3, and external docs, just invent 😉 )